cypress/e2e/invoices/cancel-invoice-created-in-current-fy-and-paid-against-previous-fy.cy.js

```diff
@@ -0,0 +1,239 @@
+import { DevTeams, TestTypes, Permissions } from '../../support/dictionary';
+import { Budgets, FiscalYears, LedgerRollovers } from '../../support/fragments/finance';
+import { Invoices, InvoiceView, InvoiceLineDetails } from '../../support/fragments/invoices';
+import { Transactions } from '../../support/fragments/finance/transactions';
+import TopMenu from '../../support/fragments/topMenu';
+import Organizations from '../../support/fragments/organizations/organizations';
+import Users from '../../support/fragments/users/users';
+import { NewOrder, Orders, OrderLines } from '../../support/fragments/orders';
+import NewOrganization from '../../support/fragments/organizations/newOrganization';
+import BasicOrderLine from '../../support/fragments/orders/basicOrderLine';
+import { StringTools, CodeTools } from '../../support/utils';
+import { INVOICE_STATUSES } from '../../support/constants';
+
+describe('Invoices', () => {
+  const date = new Date();
+  const code = CodeTools(4);
+  const fiscalYears = {
+    first: {
+      ...FiscalYears.getDefaultFiscalYear(),
+      code: `${code}${StringTools.randomTwoDigitNumber()}01`,
+      periodStart: new Date(date.getFullYear(), date.getMonth() - 1, date.getDate()),
```

Comment:
There are already out-of-the-box methods in the DateTools. You can use them.

Reply:
Fixed